dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -56,25 +57,27 @@ internal object CoreFeature {
 
     internal var packageName: String = ""
     internal var packageVersion: String = ""
+    private var customServiceName: String? = null
```

Comment:
You didn't save any logic copy. You could do : 
```kotlin
internal var serviceName: String = ""

fun initialize(…) {
    // …
    serviceName = config.serviceName ?: packageName
}

fun stop() {
    // …
    serviceName = ""
}
```

Then in all the other `XFeature`, remove the `XFeature.serviceName` properties and reference this one here.

Reply:
Oh...yes that would be better, don't know why I didn't think about this. Tks.